api4/user_test.go

```diff
@@ -4425,6 +4426,7 @@ func TestRegisterTermsOfServiceAction(t *testing.T) {
 
 	success, resp := th.Client.RegisterTermsOfServiceAction(th.BasicUser.Id, "st_1", true)
 	CheckErrorMessage(t, resp, "store.sql_terms_of_service_store.get.no_rows.app_error")
+	assert.True(t, *success)
```

Comment:
This is causing a panic. In fact, the check is false. If there is an error, success should be `nil`.

Reply:
Ah that's right. I wanted to run all the tests locally, but they just took SO LONG, and most of them just timed out after 30 mins.